src/io/file_manager.cpp

```diff
@@ -960,7 +960,8 @@ void FileManager::checkAndCreateScreenshotDir()
     m_screenshot_dir  = getenv("HOME");
     m_screenshot_dir += "/Library/Application Support/SuperTuxKart/Screenshots/";
 #else
-    m_screenshot_dir = checkAndCreateLinuxDir("XDG_CACHE_HOME", "supertuxkart", ".cache/", ".");
+    m_addons_dir = checkAndCreateLinuxDir("XDG_DATA_HOME", "supertuxkart",
+                                          ".local/share", ".stkaddons");
```

Comment:
That should be `m_screenshot_dir` not `m_addons_dir`.
Also, `.stkaddons` as the last argument is wrong.

Reply:
Fixed, thanks.